src/bridge/secrets/bootcamps/secrets.ci.yaml

```diff
@@ -32,16 +32,101 @@ edx:
   client_secret: ENC[AES256_GCM,data:geW0Fpmn3M7JG7PzObnxEE9ypC4iybCgLlaDLSDxX4qYcf+42JmnnkpVo9j3wKlB2gl/LDnDVFW0pzh3D49ctZMe25yiSw25AwtNN4PtZyvNtNb5zUxJqE/5sQ+s/1rgH3tTVLioEmzdCr3Lx6BCRvU/VKpHDwTo56UHgvQy91A=,iv:rNMbDh1z8HKJM/4JhEGbyx5zZTFt56Q6KWiLIfulB3k=,tag:3shv7fwFYuBjXQ1BVbnWgg==,type:str]
 sops:
   kms:
-  - arn: arn:aws:kms:us-east-1:610119931565:key/29397ad6-5cfd-43ef-8f43-818b19ab75e4
-    created_at: "2023-09-19T00:34:46Z"
-    enc: AQICAHiiGjYUolrtj8PCnScLM7oLAdMl8nJrLjQjnqyl1LykYgF/6P4SCK0q8JqHMKgisIw5AAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMqA6XBIT7d7oTif8YAgEQgDtWfDe/Fdjqxo5oyHW4tC55wMszR4eY8Hm4RATAco/hbHu4hq4V0QfAdlEkGFCHI8HVOsl+/JlzTh01pQ==
+  - arn: arn:aws:kms:us-east-1:610119931565:alias/infrastructure-secrets-ci
+    created_at: "2024-01-24T19:59:03Z"
+    enc: AQICAHjnbqe9AmEW1Js10nySybyuAG7Fb5E9EHUgkmqFDv7PxQEB4DJ2Z0jGx6GvQVPdV0jHAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMpGuZ9fd0L65gqBgTAgEQgDuWy+jK3LTDXiOdA+0kB7jecpdl4UAvw4u9bsUVa2FaenLVWjvUJ71TvEgNZCqf+z7UWdev41jjnK1iog==
```

Comment:
Was the issue earlier simply a matter of pointing to the key ID rather than the alias?

Reply:
Possibly? I'm not sure...